src/src/pages/capabilities/NewCapabilityDialog.js

```diff
@@ -153,7 +153,7 @@ export default function NewCapabilityDialog({
             onChange={changeDescription}
           ></TextField>
 
-          <TextField
+          {/*<TextField
```

Comment:
Is there a reason to only remove the text field and not the following invitee-boxes?
not blocking